src/irmin-pack/layered_store.ml

```diff
@@ -41,18 +41,16 @@ module Copy
 struct
   let ignore_lwt _ = Lwt.return_unit
 
-  let copy ~src ~dst ?(aux = ignore_lwt) str k =
+  let copy ~src ~dst str k =
     Log.debug (fun l -> l "copy %s %a" str (Irmin.Type.pp Key.t) k);
-    SRC.find src k >>= function
+    match SRC.unsafe_find src k with
```

Comment:
`unsafe_find` also [recomputes the hash](https://github.com/mirage/irmin/blob/master/src/irmin-pack/pack.ml#L221), maybe we can have a faster specialised find?

Reply:
Added in #1154